CMakeLists.txt

```
@@ -130,6 +136,9 @@ add_executable(
 
 target_compile_definitions(d8 PRIVATE $<${is-msvc}:_HAS_EXCEPTIONS=0>)
 target_compile_options(d8 PRIVATE ${disable-exceptions})
+if(enable-fPIC)
```

I believe this should be (here and elsewhere):
```suggestion
if(enable-fpic)
```
Because cmake variables are case sensitive.

---

yes, they are case sensitive, which is why `enable-fPIC` matches the variable passed to `cmake`: `cmake -Denable-fPIC=ON` as suggested :)